nasa-gibs/worldview
# WV-2938 notify fix

## Description
If there is only one outage notification when dismiss it, it does not show in the "i" menu. The notifications section should still appear in the "i" menu so that a user can refer to the notification again if they wanted to.

## Steps to recreate:

- Load SIT or UAT, which currently only have one "outage" type notification
- Click on the X to close the close the outage notification that appears on load
- Go to the "i" button in the upper right corner
- See error - the Notification menu item should be there so a user can refer back to the notification they dismissed earlier.

Fixes # wv-2938

## How To Test
- `git checkout wv-2938-notify fix`
- npm ci
- npm run watch
- Confirm that you are alerted to 1 notification on page load.
- Click on the X to close the close the outage notification that appears on load
- Go to the "i" button in the upper right corner
- See error no longer exists - the Notification menu item should be there; clicking on it should show the same alert you saw earlier
## Merging

Please use the `squash and merge` commit method unless each commit in your branch is vital to the commit history of main.

@nasa-gibs/worldview
